tempesta_fw/addr.c

```diff
@@ -565,3 +518,19 @@ tfw_addr_fmt(const TfwAddr *addr, char *out_buf, size_t buf_size)
 	return (pos - out_buf);
```

Comment:
tfw_addr_fmt(): double check for supported familiy, in validate_addr() and family_is_not_supported(). Please inspect all the calls of the check and remove unnecessary calls.